onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -11,8 +11,8 @@
 
 namespace onnxruntime {
 
-static const int kMaxBatchSize = 13;
-static const int kMaxWorkSpaceSize = 1 << 24;
+static const int kMaxBatchSize = 1;
```

Comment:
we don't want these statics.
Can't we have sizes as members of the provider?  (and add accessor method)
if getenv("ORT_TENSORRT_MAX_BATCH_SIZE") 
if it's null, then set it to the default value.

Reply:
done